Python/libraries/recognizers-choice/recognizers_choice/resources/english_choice.py

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# <auto-generated>
+#     This code was generated by a tool.

```

Comment:
Can we remove this auto-generated messages??

Reply:
I think the message is to prevent people from editing this, it's on all the resources!

Reply:
Great! I thought it was generated by a converter